build_tools/cmake/iree_cc_test.cmake

```diff
@@ -108,7 +108,8 @@ function(iree_cc_test)
   # We run all our tests through a custom test runner to allow temp directory
   # cleanup upon test completion.
 
-  string(REPLACE "_" "/" _PACKAGE_PATH ${_PACKAGE_NAME})
+  iree_package_ns(_PACKAGE_NS)
```

Comment:
_PACKAGE_NS is defined with iree_package_ns already above. Can you reuse it instead?

Reply:
Oh yeah, thanks! Missed that one.